src/diff/props.js

```diff
@@ -97,6 +97,12 @@ function setProperty(dom, name, value, oldValue, isSvg) {
 		let useCapture = name !== (name = name.replace(/Capture$/, ''));
 		let nameLower = name.toLowerCase();
 		name = (nameLower in dom ? nameLower : name).slice(2);
+		
+		// Fix onInput not firing on select/radio/checkbox in Edge <= 18.
+		if (name === 'input' && (
+			type === 'select' ||
+			type === 'input' && /rad|check/.test(props.type)
```

Comment:
```suggestion
             if ((name === 'input' || /rad|check/.test(name)) &&
			(type === 'select' || type === 'input')
```
This one makes the tests to pass for radio and checkboxes as well. Is this what you had in mind @developit? 😄